http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GceInterpreter.scala

```diff
@@ -158,6 +158,26 @@ class GceInterpreter[F[_]: Async: Parallel: ContextShift: Logger](
                                               None)
     } yield CreateRuntimeResponse(asyncRuntimeFields, initBucketName, None, config.gceConfig.customGceImage)
 
+  override def getRuntimeStatus(
+    params: GetRuntimeStatusParams
+  )(implicit ev: ApplicativeAsk[F, TraceId]): F[RuntimeStatus] =
+    for {
+      zoneName <- Async[F].fromEither(
```

Comment:
For other methods in `GceInterpreter` I've just been using the zone in `GceConfig`